docs/jsx.md

```diff
@@ -338,8 +367,23 @@ function SummaryView() {
 
 ---
 
-### Converting HTML
+### Tips and tricks
+
+#### Converting HTML
```

Comment:
I missed this part when I recast this, and should've clarified. Apologies for polluting the review comments with this.

```suggestion
#### Converting HTML to JSX
```